contents/blog/product-360.md

```diff
@@ -0,0 +1,126 @@
+---
+date: 2022-02-03
+title: I used to think you don't need product people. I was wrong.
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog", "Engineering"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/product-people.png
+featuredImageType: full
+---
+
+When PostHog was fresh out of YC last year, there were two things we focused on: (i) build and (ii) talk to customers. Hiring a product person felt like it would slow us down. Engineers on our team would speak to our users (also engineers) - surely they'd know what's best to prioritize? We didn't want our engineers to be "shielded" from our customers.
+
+By [PostHog's series B](/blog/15-million-series-b), our team showed me that I was totally wrong. Here's why that was the case, what we did about it, and the results.
+
+>James Hawkins is PostHog's Co-Founder & CEO. Follow him on [Twitter](https://www.linkedin.com/in/j-hawkins/).
+
+## Why I used to think this way
+
+My previous experience of product management:
+
+* Here's a list of features the enterprise sales team wants
+* How much pipeline does each of these features influence?
+* _Builds as much of the above as possible whilst trying to manage existing customer requests_
+
+Led to:
+
+* Lots of initial enterprise contracts
+* A product users find frustrating to use
+* No word of mouth growth
+* Customer churn in the long term
+
+## It starts with expectation setting
+
+<a href="https://imgflip.com/i/61pr1n"><img src="https://i.imgflip.com/61pr1n.jpg" title="made at imgflip.com"/></a><div><a href="https://imgflip.com/memegenerator">from Imgflip Meme Generator</a></div><br />
+
+Alas.
+
+Whilst we should measure what matters – often it's what is measured, is what ends up mattering.
+
+Revenue is pretty easy to measure and feels important. A B2B SaaS company will therefore default into prioritizing this. That feels intuitive - your company will die without money.
+
+And you should sell more to get more revenue, right?
+
+<div style="width:360px;max-width:100%;"><div style="height:0;padding-bottom:65%;position:relative;"><iframe width="360" height="234" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameBorder="0" src="https://imgflip.com/embed/61rx1x"></iframe></div><p><a href="https://imgflip.com/gif/61rx1x">via Imgflip</a></p></div>
```

Comment:
I'm inclined to remove the second meme. Two in the space of a paragraph or two feels a bit much.

Reply:
agreed